backend/api/models/Promotion.model.js

```diff
@@ -1,14 +1,15 @@
 const { Schema, model } = require('mongoose')
     
 const promoSchema = Schema({
-    isbn: { type: Number, required: true },
```

Comment:
Can you add a start date? You can just set the default value to Date.now(). Also I think you can remove the title.

Reply:
I can add a startdate -- for the title my thought was customers will actually have to type the promotion in at some point.

Reply:
Ohhh. I thought it was the title of a book. Okay. Nvm on that one